### ISSUE TITLE
gathering and locating centers

### ISSUE THREAD

**Issue by OlivierJM** [NatashaChuza/Tipezane#2]
We will basically need a list of all centers and a way to collect these centers

**Comment by OlivierJM** [comment]
we can start with a JSON file that contains the known centers and later we can make this more dynamic.

**Comment by OlivierJM** [comment]
the structure of a center could be something like this

```js
{
  name: '',
  location: [latitude, longitude ],
  town: '',
  contact: '',
  personInCharge: ''
}

```
This can be changed, just for a start

**Comment by NatashaChuza** [comment]
cool, I'm following. Is there like an api that already has this information currently?

**Comment by OlivierJM** [comment]
I don't think there is any but heyy we can make one 😃

**Comment by OlivierJM** [comment]
we can start with the information we know and later we can provide users with a way of adding more centers(we would need to verify them )

**Comment by NatashaChuza** [comment]
cool. I don't  know any orphanages though

**Comment by NatashaChuza** [comment]
And also, how does the app work exactly, does one enter the name of the orphanges in a search box then app shows a map like interface to the orphanage?

**Comment by NatashaChuza** [comment]
or, the app should show the nearby orphanages when one logs in

**Comment by OlivierJM** [comment]
I know like 3 in Lusaka, but I believe there is more, in the future we could have a section for hospitals maybe 🤷🏾‍♂️
Both options are possible, the initial idea was that you don't even know the name of any orphanage but you want to send them something or maybe pay them a visit, so in this case the second option will be better, you open the app then it shows you the nearby centers with a listing of others as well

**Comment by OlivierJM** [comment]
The nearby could be a bit hectic to implement if the centers aren't properly mapped on google maps

**Comment by NatashaChuza** [comment]
so what would we do if that were the case?

**Comment by OlivierJM** [comment]
in that case we would rely on the exact location, that is the latitude and longitude which will need to picked up from the actual centers.

**Comment by OlivierJM** [comment]
I checked google maps doesn't identity orphanages yet in Zambia, it could have been very easy

**Comment by NatashaChuza** [comment]
But how do we get information about the centers, central statistical office?

**Comment by OlivierJM** [comment]
🤔 not an easy one
I think these will have to be community curated

**Comment by OlivierJM** [comment]
we can give ourselves a bit of sometime to think about this, so that we come up with a better solution

**Comment by NatashaChuza** [comment]
okay then let's deal with what we can for now

**Comment by OlivierJM** [comment]
Yeah agreed.

**Comment by NatashaChuza** [comment]
so the site will have orphanages, community centers and the like, if someone wants to say donate money to an orphange, they can do that online right?

**Comment by OlivierJM** [comment]
I think for donate, we can have(If possible) contacts for people in charge at those orphanages then add them there so that someone can contact them, we might need to think on a secure way for this to avoid any scams.

**Comment by NatashaChuza** [comment]
and I know the app will mainly be about locating orphanges and community centers, but I think we should have a way to broadcast what is happening in a community, how people can help and volunteer

**Comment by OlivierJM** [comment]
That would be great super great

**Comment by NatashaChuza** [comment]
you're right about the scams

**Comment by OlivierJM** [comment]
Yeah let's keep it for future version when we have stabilized how we will be getting centers locations and details

**Comment by OlivierJM** [comment]
But this one is great 
> and I know the app will mainly be about locating orphanges and community centers, but I think we should have a way to broadcast what is happening in a community, how people can help and volunteer

**Comment by NatashaChuza** [comment]
what else does our app need

**Comment by NatashaChuza** [comment]
There's no need for signing up and what not right

**Comment by OlivierJM** [comment]
we wont need signup for now

**Comment by OlivierJM** [comment]
If this comes up well we can submit it to the upcoming facebook community challenge hackathon(June I think)

**Comment by NatashaChuza** [comment]
Then we're set?

**Comment by NatashaChuza** [comment]
> If this comes up we can submit it to the upcoming facebook community challenge hackathon(June I think)

for facebook developers circle?

**Comment by OlivierJM** [comment]
I think we basically have most of what need to get started with, was trying to think about a layout but nothing good yet.

**Comment by OlivierJM** [comment]
> for facebook developers circle?

Yes

**Comment by NatashaChuza** [comment]
> > for facebook developers circle?
> 
> Yes

dope

**Comment by NatashaChuza** [comment]
For the UI?

**Comment by OlivierJM** [comment]
yeah a layout for the UI, any suggestions ? 

We can look into some inspirations and gather ideas, although this should be simpler and easy to access, for now am thinking of a good list(it could be cards, or any other listing view) and a map somewhere.

**Comment by NatashaChuza** [comment]
I think the UI should be very minimal and informative

**Comment by NatashaChuza** [comment]
It should sort of guide the user on what to do and how to do it

**Comment by OlivierJM** [comment]
> It should sort of guide the user on what to do and how to do it  

Definitely, I am stepping away for a while, let's keep discussing this and get started sooner

**Comment by NatashaChuza** [comment]
where are you going?

**Comment by OlivierJM** [comment]
Heading to Matero

**Comment by NatashaChuza** [comment]
shame, I was having fun. I'll try coming up with a mock up in the mean time then

**Comment by OlivierJM** [comment]
Trust me I enjoyed this too and this was and will continue to be a productive conversation, a mock-up will help a lot, looking forward to this. Thanks a lot
